docs/reference/indices/split-index.asciidoc

```diff
@@ -121,11 +121,15 @@ newly split index.
 A split operation:
 
 . Creates a new target index with the same definition as the source
-  index, but with a larger number of primary shards.
+  index, but with a larger number of primary shards. Created shards allocate
+  to the same nodes as their correlating source primary shard, so it must
+  already have sufficient disk to host the copy of the data.
```

Comment:
After consulting @tlrx (thanks!) indeed if the shards are on different nodes, the new index shards will be allocated and split on the nodes of their corresponding shards.

We should probably also clarify the above in the following sentence later in the document:

> The node handling the split process must have sufficient free disk space to accommodate a second copy of the existing index.

to something like

> The nodes handling the split process must have sufficient free disk space to accommodate a second copy of the original shards.